web/ui/navbar/other.svelte

```diff
@@ -32,8 +32,22 @@
     icon={mdiFileTree}
     secondary
   />
+
+  <NavbarItem
+    name={$t.export}
+    current={$router.route === 'export'}
+    href={getURL('export')}
+    secondary
+  />
 {/if}
 
+<NavbarItem
```

Comment:
You can use this icons:
https://pictogrammers.com/library/mdi/icon/export/
https://pictogrammers.com/library/mdi/icon/import/